src/program/lwaftr/setup.lua

```diff
@@ -24,9 +26,13 @@ function lwaftr_app(c, conf)
               { mtu=conf.ipv4_mtu })
    config.app(c, "fragmenterv6", ipv6_apps.Fragmenter,
               { mtu=conf.ipv6_mtu })
+   config.app(c, "ndp", ipv6_apps.NDP,
+              { src_ipv6 = conf.aftr_ipv6_ip, src_eth = conf.aftr_mac_b4_side,
+                dst_eth = conf.b4_mac, dst_ipv6 = conf.next_hop_ipv6_addr,
+                all_ipv6_addrs = conf.preloaded_binding_table:get_br_addresses() })
 
    local preprocessing_apps_v4  = { "reassemblerv4" }
-   local preprocessing_apps_v6  = { "reassemblerv6" }
+   local preprocessing_apps_v6  = { "reassemblerv6"}
```

Comment:
Yep, typo. Adrian'll be changing this for the changes in his ICMP patch anyhow, I think, but reverted. :)